src/types/widget-data/legend.ts

```diff
@@ -66,3 +66,13 @@ export type PathLegendSymbolOptions = BaseLegendSymbol & {
      * */
     width?: number;
 };
+
+export type SymbolLegendSymbolOptions = BaseLegendSymbol & {
+    /**
+     * The pixel size of the symbol for series types that use a path in the legend
+     * It needs to be casted to d3 size before using in d3.symbol
```

Comment:
This will go into the documentation, so I would suggest describing it in more detail. What does "needs to be casted to d3 size" mean?